Django-CoWork-Connect/coapp/models.py

```diff
@@ -1,3 +1,10 @@
 from django.db import models
-
-# Create your models here.
+class Desk(models.Model):
+    desk_type = models.CharField(max_length=255)
+    monitor = models.BooleanField(default=False)
```

Comment:
Może lepiej tutaj monitro_number = integer field I jak nie ma monitora to pisać 0 jak jest to 1 jak są dwa to 2 itd